src/lib/block/kuznyechik/kuznyechik.cpp

```diff
@@ -0,0 +1,4394 @@
+/*
+* GOST R 34.12-2015: Block Cipher "Kuznyechik"
```

Comment:
Please add a Copyright notice from yourself following the usual convention

Reply:
Done